esp-hal-embassy/MIGRATING-0.3.md

```diff
@@ -1,8 +1,6 @@
-Migration Guide from 0.3.x to vNext
-====================================
+# Migration Guide from 0.3.x to v0.4.x
 
-Initialsation
--------------
+## Initialsation
```

Comment:
```suggestion
## Initialization
```

Pains me to use that version :D, but we may as well be consistent with the rest of our docs